BMM.Core/Models/Themes/Theme.cs

```diff
@@ -4,6 +4,10 @@ public enum Theme
     {
         Light,
         Dark,
-        System
+        System,
+        GoldenLight,
```

Comment:
I don't think it looks golden.
But Randi needs to decide on the rewards to make proper progress here.